tests/invoke_sandbox.rs

```diff
@@ -26,3 +26,20 @@ fn invoke_token() -> Result<(), Box<dyn std::error::Error>> {
 
     Ok(())
 }
+
+#[test]
+fn source_account_exists() -> Result<(), Box<dyn std::error::Error>> {
+    let tmp = TempDir::new().unwrap();
+    let ledger = tmp.child("ledger.json");
+
+    let mut cmd = Command::cargo_bin("soroban")?;
+    cmd.arg("invoke");
+    cmd.arg("--ledger-file").arg(ledger.as_os_str());
+    cmd.arg("--id").arg("1");
+    cmd.arg("--wasm")
+        .arg("target/wasm32-unknown-unknown/test-wasms/test_invoker_account_exists.wasm");
```

Comment:
Optional: Since the test doesn't explicitly enforce building the test contracts, it may be a good idea to give a firendly/specific error if the file does not exist.

Reply:
Good idea. I added more descriptive panic message in f4b29b2. Note that this is still a large wall of text, because any test failure usually results in a pretty big wall of text, such is the Rust way as far as I can tell.